src/DependencyInjection/EightPointsGuzzleExtension.php

```diff
@@ -91,13 +92,21 @@ public function load(array $configs, ContainerBuilder $container)
             // set service name based on client name
             $serviceName = sprintf('%s.client.%s', $this->getAlias(), $name);
             $container->setDefinition($serviceName, $client);
+
+            // Allowed only for Symfony 4.2+
+            if (\method_exists($container, 'registerAliasForArgument')) {
+                if ('%eight_points_guzzle_bundle.http_client.class%' !== $options['class']) {
```

Comment:
I'm not sure about this comparison. Variable `$option['class']` by default has value `GuzzleHttp\Client`. Current implementation always will give true.

Reply:
Default value in (configuration)[https://github.com/8p/EightPointsGuzzleBundle/blob/fb0c746cc444523b4a887f2cb952dd45aac90bce/src/DependencyInjection/Configuration.php#L95] is '%eight_points_guzzle_bundle.http_client.class%' and in my set-up, it still has that value at given point. However, it's good to write a test case to make sure it is correct. I'll do that first.